WAMPClient.spec.js

```diff
@@ -258,3 +251,7 @@ describe('WAMPClient', function () {
 		});
 	});
 });
+
+after(() => {
+	clock.restore();
+});
```

Comment:
I much prefer to have these hooks all together at the start, but maybe we don't have a standard for this yet.